### ISSUE TITLE
Licence

### ISSUE THREAD

**Issue by groue** [Zewo/Sideburn#1]
Hello,

[The licence of GRMustache.swift](https://github.com/groue/GRMustache.swift/blob/v0.11.0/LICENSE) says:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Sideburn still contains substantial portions of GRMustache.swift.

Will you please update your Licence, and replace it with the content of https://raw.githubusercontent.com/groue/GRMustache.swift/v0.11.0/LICENSE

Thanks in advance!

**Comment by paulofaria** [comment]
Sorry, mate! I retained all the License headers in all files. I thought that was all. So, should I replace your license in my LICENSE file or append it? I still don't get how this licensing stuff works.

**Comment by paulofaria** [comment]
> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

**All** source files but the LICENSE have your header. This should count as a substantial portion of the software, shouldn't it? If you feel I have to change the LICENSE as well, that's no problem at all.

**Comment by groue** [comment]
Well, I claim GRMustache.swift copyright, so please do update your License file, yes.

Now, understand me: I'm very happy that you've built a linux-ready Mustache renderer on top of GRMustache.swift. I hope Sideburn will be useful, and not only to you. We may even work together in the future, I don't know. Today, I'd appreciate a clear copyright since Zendo hasn't diverged enough.

**Comment by paulofaria** [comment]
No worries mate, I changed it! Check if it's right now. I was just trying to understand the correct interpretation of the license statement. It seems I was getting it wrong. If you have any plans to make GRMustache.swift linux compatible let me know. I forked it because this is a must-have feature and I can't force you to do it. However, if you're willing to do it, I don't see the point in having this fork. So, just let me know if you do, right? Thanks mate!
